lib/bpk-component-calendar/src/common-types.js

```diff
@@ -116,7 +136,17 @@ export const commonPropTypes = {
   maxDate: datePropType,
   onChangeSelectedDates: PropTypes.func,
   selectedDates: selectedDatesPropType,
-  selectionType: PropTypes.oneOf(Object.keys(SELECTION_TYPES)),
+  selectionType: PropTypes.oneOfType([
+    PropTypes.shape({
+      _type: PropTypes.oneOf(['single']),
+    }),
+    PropTypes.shape({
+      _type: PropTypes.oneOf(['range']),
+    }),
+    PropTypes.shape({
+      _type: PropTypes.oneOf(['multiple']),
+    }),
+  ]),
```

Comment:
I know this is WIP, but why not use this
```suggestion
  selectionType: PropTypes.shape([
    _type: PropTypes.oneOf(['single', 'range', 'multiple']),
  }),
```

Reply:
Right now that would work, but when we add the a11y strings these will all be different even though they all have the `_type` field.

Reply:
Oh it's because each one is gonna have different additional strings.

Reply:
I've updated this now